src/Confgen.php

```diff
@@ -103,7 +103,10 @@ private function processTemplatesData(array $templates, $data)
 
         // let's reload all the files after (successfully) writing all of them
         foreach ($commands as $command) {
-            $this->executor->executeCommand($command);
+            $code = $this->executor->executeCommand($command);
+            if ($code !== 0) {
+                throw new \RuntimeException('Unable to execute "' . $command . '"');
```

Comment:
Not sure about error message here. Does it make sense to include exit code in output?

Also, can you add this expected behavior to the documentation?

Reply:
Yes, make sense to return the error code!
Ok, will add it to documentation